democracy_club/templates/polling-stations/home.html

```diff
@@ -10,72 +11,78 @@ <h1>Polling station finder</h1>
-* Embeddable in your web site
-* Fully customisable and white-label versions on request
-* Almost any data format supported
-* High performance and scalability ensures uptime on the busiest of elections
-* Fully featured API
-
-
-## Data changes
-
-We only use data sent to us for the elections you tell us it's valid for.
-If we're not sure we don't re-use data between elections, and we will email you asking
-for updates.
-
-
-For last minute changes, including on polling day, you can email us and we will make
-changes to the data on the live site. In the worse case we can remove all your data from
-the site very quickly, just get in touch.
-
-
-## Data restrictions
-
-We don't need personal data to run the polling station finder but we do need
-UPRNs or data that from some of OS's products depending on the type of data you are
-able to provide us with.
-
-We have an agreement with The Electoral Commission under the Public Sector Mapping Agreement
-that allows us to receive data that's in AddressBase including UPRNs.
-
-If you send us geospatial data that's created from OS's BaseMap then this can be made public
-under ‘presumption to publish’.
-[You can read more about this here](http://www.ordnancesurvey.co.uk/docs/public-disclosures/presumption-to-publish-guidance-march-2015.pdf).
-
-We suggest that you publish Geo data as a WFS layer from your GIS software.
-
-
-## Using the finder on your website
-
-If you provide data to us then you can embed out polling station directly on
-your website, free of charge.
-
-You'll need to add a bit of code to your site.
-[Find out how]({% url "polling_embed_code" %}).
-
+Our polling station finder can be used by voters wherever they look.
+On our own <a href="https://whocanivotefor.co.uk">WhoCanIVoteFor.co.uk</a>
+website, their local council website, their local news website or on
+The Electoral Commission's YourVoteMatters website.
+
+Voters don’t need to know which ward or constituency they’re in or who manages their elections —
+we just get them the answer they’re searching for, in seconds.
+
+## Plans and pricing for local authorities
+
+
+<div class="card">
+<h3>Basic: free</h3>
+
+<ul>
+<li>Finds the polling station assigned to a property</li>
+<li>Support postcosts that overlap polling district boundaries</li>
+<li>Data import from your EMS*</li>
+<li>Data included on the Electoral Commission’s YourVoteMatters lookup as well as other sites we partner with</li>
+<li>Embeddable widget on your website</li>
+</ul>
+<a class="link-button" href="{% url 'polling_data_upload' %}">Upload data now</a>
+</div>
+
+<div class="card highlight_card">
+<h3>Pro: £450 per year</h3>
+
+All basic feautres, plus:
+<ul>
+<li>Coverage for all scheduled elections, and all national elections</li>
+
+<li><a href="https://www.digitalmarketplace.service.gov.uk/g-cloud/services/505951175500567">Approved on the Government’s Digital Marketplace (G-Cloud 10)</a></li>
+<li>Personal contact for support queries</li>
```

Comment:
"Named" instead of personal?

Reply:
A named contact and direct dial number for support queries